docs/src/pages/index/+Page.tsx

```diff
@@ -0,0 +1,13 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { allComponents } from 'components/content';
+export { Page };
+
+function Page() {
```

Comment:
```suggestion

export const Page = () => {
```
Code style of the codebase